### ISSUE TITLE
BUG: TS access modifier: static vs instance properties

### ISSUE THREAD

**Issue by AdamWillden** [MeirionHughes/aurelia-template-lint#159]
```ts
import { observable } from 'aurelia-framework';
export class Thing {
  private static zIndex: number = 1;

  @observable public expanded = false;
  public zIndex: number = 1;

  public toggle() {
    this.expanded = !this.expanded;
  }

  private expandedChanged(expanded: boolean) {
    if (expanded) {
      this.zIndex = DmaMiniStatus.zIndex++;
    }
  }
}
```
```html
<template css="z-index: ${zIndex}">
  <button type="button" click.delegate="toggle()">${expanded ? 'Hide' : 'Peek'}</button>
  <div if.bind="expanded">
    <p>Hello!</p>
  </div>
</template>
```

I get the following error which appears to be picking up the static property rather than the instance property of the same name: `[11:58:29] WARNING: field 'zIndex' in type 'Thing' has private access modifier Ln 1 Col 1 \src\thing.html`

**Comment by MeirionHughes** [comment]
published: 0.9.22

**Comment by AdamWillden** [comment]
@MeirionHughes thank you. Very impressed at the quick turn around. Keep up the good work :-)
